src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -13,6 +13,9 @@ interface Props {
   ownerName: string;
   sellerId: string;
   sellerName: string;
+  progressBarColor: string;
+  extraInfo: { [prop: string]: any }
```

Comment:
Where is this being defined/where is it coming from? iirc @juliexxia's PR hasn't merged yet

Reply:
yeah, I'm grabbing it from SellerPage and passing it in as extraInfo = {seller.extra_seller_info} as a standin. I based it off how location model got loaded with the seller object and was assuming it'll be the same with the extra info model. My bad if I should've held off, I can change once the model is merged.

Reply:
Ah got it. I think the extra info will be directly on the seller model: https://github.com/sendchinatownlove/ruby/pull/29

Not like location which is a separate object. But no this is totally good!

Reply:
As long as the model in the backend stays on the seller model, you can model it however you want in the frontend. I think encapsulating it like that is fine :)

Reply:
Got it!

Reply:
sendchinatownlove/ruby#29 is submitted! extra info is directly on the seller model like Justin said, not in its own model.

Reply:
Ty Julie! Will make changes on my end

Reply:
Hi @juliexxia, I'm getting an error pulling+ migrating after the latest merge saying there's an undefined model called extra_seller_infos. I haven't picked up enough/any rails to fix the issue on my end, do you have any suggestions on what I can do?
<img width="1680" alt="Screen Shot 2020-04-21 at 12 46 46 AM" src="https://user-images.githubusercontent.com/47200207/79889932-b4511080-83cc-11ea-98d7-4b23c5efc02c.png">

Not sure if I was the only one getting it, so didn't post on slack

Reply:
you have to migrate your database because she added new fields to the Sellers table.
```
rails db:create # only if you haven't done this before
rails db:migrate # to run a migration
```

Reply:
I got that error after dropping ->recreating->migrate. It must be on my end only, I'll do some digging
<img width="1321" alt="Screen Shot 2020-04-21 at 12 56 06 PM" src="https://user-images.githubusercontent.com/47200207/79891973-c2ecf700-83cf-11ea-85fe-f60b74f5b2ac.png">

Reply:
Sent you a message on Slack